exploring_UV_extinction_curves_pt1/downloading_UV_data_pt1.ipynb

```diff
@@ -0,0 +1,1433 @@
+{
```

Comment:
<p>Here's some good info that would be helpful for the reader in the introduction; giving a little more information about what we're going to do. We want to find the spectra of two stars, one reddened and one unreddened. What is their relationship to one another... or is there one? (Other than that they are of the same spectral type?) Were these chosen from a catalog, if so which one (e.g. you could cite it)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>